pyproject.toml

```diff
@@ -0,0 +1,95 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pyproject.toml -- Ensembl GenomIO project configuration
+
+[build-system]
+requires = [
+    "setuptools",
+    "setuptools-scm",
+    "wheel"
+]
+build-backend = "setuptools.build_meta"
+
+
+## Generic configuration
+[project]
+name = "ensembl-genomio"
+version = "0.1" # should be assigned dynamically in the future
+requires-python = ">= 3.7"
```

Comment:
Can I ask any particular reason to use 3.7+ instead of 3.8+ ? I noticed some dep issues when importing latest version of numpy for e.g. which required 3.8

Reply:
not 3.8, though.